### DISCUSSION TITLE
NAO Walking example

### DISCUSSION THREAD

**Discussion by NoePity2** [softbankrobotics-research/qibullet#83] [Q&A]
Hello.

Thank you very much for your project. Is there any example of a Nao robot walking?

Best regards.

**Comment by mbusy** [comment]
Hi, you're welcome! Please see #47 for more explanations. To sum up, the NAO walking gif of the README has been generated by mimicing the joints of a real NAO walking in real life. At the moment there's no example that would allow you to reproduce that behaviour in the project.

In #47, I describe 2 potential solutions, using a walk engine, or using the solution described by robogeekcanada [here](https://github.com/robogeekcanada/pybullet-tutorials/blob/main/nao_motions.py).

I hope this will help you!
